.buildkite/pipeline.yml

```diff
@@ -15,7 +15,7 @@ steps:
   - label: ":docker: build proxy docker image"
     trigger: "neon-proxy"
     build:
-       branch: "${PROXY_BRANCH:-develop}"
+       branch: "122-estimate_gas_on_deploy_a_contract"
```

Comment:
Revert this

Reply:
Reverted. But it must be borne in mind that without these changes, some checks will not pass.